api/src/filters/validation-error.filter.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    ExceptionFilter,
+    Catch,
+    ArgumentsHost,
+    BadRequestException,
+    HttpStatus
+} from "@nestjs/common";
+import { Response } from "express";
+
+interface IExceptionResponseValues {
+    statusCode: HttpStatus.BAD_REQUEST;
+    message: string | string[];
```

Comment:
So message or messages?

Reply:
i must think about it 1 more time

Reply:
```ts
getResponse(): string | object;
```

that is type from function which i use:

```ts
const exceptionResponse =
            exception.getResponse() as IExceptionResponseValues;
````

IDN how to do it, because only "message" prop this function return in object, but it's can be string or Array with many strings, i can't do it in other way, anyway i create it a little more precise, cause i type this object, in that library, they leave it as plain "object" type

Reply:
I think this solution is good